samples/net/sockets/echo_server/boards/nucleo_g431rb.overlay

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
This can be in the same commit as the sample board config.

Reply:
Ok, will make those two files as a separate commit.